src/sentry/static/sentry/app/views/organizationEvents/index.jsx

```diff
@@ -41,7 +41,7 @@ class OrganizationEventsContainer extends React.Component {
     const values = {
       project,
       environment,
-      period: query.period || (hasAbsolute ? null : '7d'),
+      period: query.statsPeriod || (hasAbsolute ? null : '7d'),
```

Comment:
If `statsPeriod` is what the API expects, should we just use that everywhere instead of `period`? or is that too baked into a bunch of other components?

Reply:
Currently the parameter name is inconsistent within our API endpoints, but I'm not sure if we want to continue using the health endpoint I'm currently using for the chart (which accepts `period`). I'm going to leave it as is for now, but I agree that we should make these consistent.